mreg/api/v1/serializers.py

```diff
@@ -243,7 +245,9 @@ def validate(self, data):
 
         def _get_ip(attr):
             ip = data.get(attr) or getattr(self.instance, attr)
-            if isinstance(ip, str):
+            # ip should be an object at this point, and it's never a string.
+            # TODO: Check data path and remove?
+            if isinstance(ip, str):  # pragma: no cover
```

Comment:
Inner functions are really annoying to test. Ideally we'd want to test the method itself, but since that's not doable we end up having to produce input to suit the outer method. An option here is to refactor `_get_ip` into something we can test on its own.

Reply:
@terjekv I'm fine with any of those 2 alternatives. Would try to produce input for the outer method first, and see if that becomes too complicated.

Reply:
Well, pruned in https://github.com/unioslo/mreg/pull/484/commits/014e25b6030a05adf9622694539b92a38f71a2b1

Reply:
Note that this is actually never objectified at all. The IP address is always a string at this point. It seems to have been validated as an IP address though, just never coerced into an object or type.